python/pylibcudf/pylibcudf/datetime.pyx

```diff
@@ -33,11 +58,227 @@ cpdef Column extract_year(
     cdef unique_ptr[column] result
 
     with nogil:
-        result = cpp_extract_year(values.view())
+        result = cpp_extract_year(input.view())
+    return Column.from_libcudf(move(result))
+
+cpdef Column extract_month(
```

Comment:
These (the individual single component extraction methods) will be deprecated in the c++ layer for 24.12 (I should have done this already). Should we still add them, and if so, should they be deprecated from the outset?